jetty-server/src/main/config/modules/test-keystore.mod

```diff
@@ -9,13 +9,10 @@ ssl
 ssl
 
 [files]
-basehome:modules/test-keystore/keystore|etc/test-keystore
+basehome:modules/test-keystore/keystore.p12|etc/test-keystore.p12
```

Comment:
can we just call the file either keystore.p12 or test-keystore.p12 and not rename during the move?

Reply:
Okay.